src/main/io/beeper.c

```diff
@@ -55,13 +55,18 @@ PG_REGISTER_WITH_RESET_TEMPLATE(beeperDevConfig_t, beeperDevConfig, PG_BEEPER_DE
 #endif
 #ifdef BEEPER
 #define BEEPER_PIN      BEEPER
+#define BEEPER_PWM_FREQ 0
+#elif defined(BEEPER_PWM)
+#define BEEPER_PIN      BEEPER_PWM
```

Comment:
generally there is hardware-independent beeper concept of something generating sound (BEEPER) and hardware implementation - ONOFF/PWM/ESC/VTX/...
Whato do you think about adding BEEPER_ONOFF for existing hardware-dependent code and BEEPER_PWM for this PR HW driver? And use BEEPER for higher levels?